handlers/reporter_test.go

```diff
@@ -103,6 +106,10 @@ var _ = Describe("Reporter Handler", func() {
 		Expect(latency).To(BeNumerically(">", 0))
 		Expect(latency).To(BeNumerically("<", 10*time.Millisecond))
 
+		Expect(fakeReporter.CaptureGorouterTimeCallCount()).To(Equal(1))
+		gorouterTime := fakeReporter.CaptureGorouterTimeArgsForCall(0)
+		Expect(gorouterTime).To(BeNumerically(">", 0))
```

Comment:
Can you add a second test kind of like on line 107? To make sure it is a small value?